packages/rule-http-compression/src/rule.ts

```diff
@@ -259,8 +259,17 @@ export default class HttpCompressionRule implements IRule {
         };
 
         const checkBrotli = async (resource, element) => {
-            const { contentEncodingHeaderValue, rawResponse, response } = await getNetworkData(resource, { 'Accept-Encoding': 'br' });
+            let contentEncodingHeaderValue;
+            let rawResponse;
+            let response;
 
+            try {
+                ({ contentEncodingHeaderValue, rawResponse, response } = await getNetworkData(resource, { 'Accept-Encoding': 'br' }));
+            } catch (err) {
+                await context.report(resource, element, 'Error fetching the content');
```

Comment:
Maybe give more information in what situation the request failed.

e.g.  `Error fetching the content when requesting the content compress with Brotli`

Reply:
ok!